services/app-web/src/components/Modal/Modal.test.tsx

```diff
@@ -34,18 +34,19 @@ describe('Modal', () => {
             </div>
         )
 
-        await waitFor(() => handleOpen())
-
-        expect(screen.getByRole('dialog')).toHaveClass('is-visible')
-        expect(
-            screen.getByRole('heading', { name: 'Test Modal Title' })
-        ).toBeInTheDocument()
-        expect(
-            screen.getByRole('button', { name: 'Cancel' })
-        ).toBeInTheDocument()
-        expect(
-            screen.getByRole('button', { name: 'Submit' })
-        ).toBeInTheDocument()
+        handleOpen()
```

Comment:
I don’t understand why handleOpen is this way. seems strange that we wrapped it it waitFor before, is `toggleModal` async?

Reply:
> seems strange that we wrapped it it waitFor before  

This also seemed strange to me, so I removed the waitFor and tests still passed, so I kept it out.  toggleModal [is not](https://github.com/trussworks/react-uswds/blob/main/src/components/Modal/utils.ts#L29) async.